### DISCUSSION TITLE
问题

### DISCUSSION THREAD

**Discussion by codeonline26** [dreamhunter2333/cloudflare_temp_email#318] [Q&A] [answered]
大佬我有几个问题
1.Code 400: Failed to send mail Mailchannels error: 500 {"errors":["Failed to send email: 550 5.7.1 This sender is not authorized to send from tms6.xyz. See https://bit.ly/domain-lockdown. cfid=tms6.xyz"]}是什么意思（发送邮件时显示）？
2.Code 400: Please enable SEND_MAIL to use verifiedAddressList怎么打开“SEND_MAIL”配置？
3.# send_email = [
#    { name = "SEND_MAIL" },
# ]有什么用？怎么用？（我的网址https://temp-mail.tms6.xyz/
测试账号tmptest@tms6.xyz
还有TOKEN是eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZGRyZXNzIjoidG1wdGVzdEB0bXM2Lnh5eiIsImFkZHJlc3NfaWQiOjF9.cbHb8VjG3gtextfPlQVzu0JVqxGKEzPYNbigFKjMnFk）

**Comment by dreamhunter2333** [comment]
此功能是使用cf 的邮件功能，只能向已经在 cf 认证的邮箱发送邮件，可以不使用第三方，用来做通知或者转发

https://developers.cloudflare.com/email-routing/email-workers/send-email-workers/

**Comment by dreamhunter2333** [comment, marked as answer]
> 1.Code 400: Failed to send mail Mailchannels error: 500 {"errors":["Failed to send email: 550 5.7.1 This sender is not authorized to send from tms6.xyz. See https://bit.ly/domain-lockdown. cfid=tms6.xyz"]}

这个看起来是配置错误，参考此处配置 DNS，https://temp-mail-docs.awsl.uk/zh/guide/config-send-mail.html

推荐使用 resend，[Mail Channels 免费电子邮件发送 API 将于2024年6月30日结束](https://support.mailchannels.com/hc/en-us/articles/26814255454093-End-of-Life-Notice-Cloudflare-Workers)

**Comment by codeonline26** [comment]
谢谢大佬，还有如何关掉注册时的人机验证啊？

**Reply by dreamhunter2333** [reply]
不配置 这几个参数即可
```toml
# Turnstile 人机验证配置
# CF_TURNSTILE_SITE_KEY = ""
# CF_TURNSTILE_SECRET_KEY = ""
```

**Reply by codeonline26** [reply]
关不掉，好像还是会显示请完成人机验证

**Reply by dreamhunter2333** [reply]
> 请完成人机验证

是否开启了 邮箱验证，邮箱验证目前必须进行人机验证

**Reply by codeonline26** [reply]
o知道了谢谢（这是个BUG吗？ (小声)）

**Reply by dreamhunter2333** [reply]
> o知道了谢谢（这是个BUG吗？ (小声)）

最开始这么写的，防止刷邮件，我想想要不要去掉

**Reply by dreamhunter2333** [reply]
> o知道了谢谢（这是个BUG吗？ (小声)）

main 分支已修改
https://github.com/dreamhunter2333/cloudflare_temp_email/commit/9448b3c754b632d292fab08380f1cf98213662b5
